portfolio/src/main/webapp/script.js

```diff
@@ -40,6 +40,29 @@ function showProjects(){
 
 }
 
+// Display pop up form
+
+function showForm(){
+    const formContainer = document.getElementById('contact-form-container');
+    const right = document.getElementById('right-content');
+
+
+    if (formContainer.style.display === 'none'){
+        formContainer.style.display = 'block';
```

Comment:
The Google style guide recommends indenting with 2 spaces instead of 4: [saved](http://google3/java/com/google/commerce/bizbuilder/frontend/boq/editor/commonweb/common/eventcontract/events.js?q=saved)

I'd also consider removing one of the newlines on line 48 and 49, and maybe both of the newlines on lines 61 and 62. Some vertical whitespace is good for readability, but too much gets distracting, imho.